src/config.ts

```diff
@@ -31,10 +31,6 @@ function loadOptionalEnvVar(envVarName: string): string | undefined {
 }
 
 function loadRequiredEnvVar(envVarName: string): string {
-  if (process.env.NODE_ENV === "test") {
```

Comment:
Removed because all the configuration variables will now be mocked for all tests. See `vitest-setup.ts` file.